pkg/informers/service/informer_service.go

```diff
@@ -145,3 +147,33 @@ func (s *ServiceImpl) GetNSTemplateTier(name string) (*toolchainv1alpha1.NSTempl
 	}
 	return tier, err
 }
+
+func (s *ServiceImpl) BannedUsersByEmail(email string) ([]toolchainv1alpha1.BannedUser, error) {
```

Comment:
Maybe we want to name it `ListBannedUsersByEmail` just for consistency with other functions but I'm fine with keeping it as is.
```suggestion
func (s *ServiceImpl) ListBannedUsersByEmail(email string) ([]toolchainv1alpha1.BannedUser, error) {
```

Reply:
done in b3a2d59a659a23d1f72253747637ff28104e56d1